eth2util/network.go

```diff
@@ -54,7 +54,7 @@ var (
 	Holesky = Network{
 		ChainID:               17000,
 		Name:                  "holesky",
-		GenesisForkVersionHex: "0x00017000",
+		GenesisForkVersionHex: "0x01017000",
```

Comment:
Can you please add a link to where this is mentioned?

Reply:
https://github.com/eth-clients/holesky/blob/main/custom_config_data/config.yaml#L11C23-L11C33